lib/segment/src/segment.rs

```diff
@@ -1220,6 +1264,83 @@ impl Drop for Segment {
     }
 }
 
+fn move_strip_prefix_all(dir: &Path, dest: &Path, prefix: &Path) -> OperationResult<()> {
```

Comment:
Could you please give this function some doc-string with example of what it is supposed to do?

Reply:
Done.